en/docs/learn/configuring-multifactor-authentication.md

```diff
@@ -0,0 +1,167 @@
+# Configuring Mutli-Factor Authentication
+
+Multi-factor authentication is an authentication mechanism that enhances security by granting access to users only after they have successfully passed two or more layers of authentication to prove their identity. For example, in addition to providing a username and password to login, an application can be configured to request users to provide a one time password (OTP) or fingerprint verification as an extra authentication step. 
+
+## Scenario
+
+A taxi company called "Pickup" uses an application called "Pickup Dispatch", which is used by their drivers to accept hires. Lately, Pickup has noticed that users who are not drivers employed at Pickup have been logging in via driver accounts. 
+
+To ensure that only their own drivers can log into the application, Pickup decided to enhance security by configuring multi-factor authentication. After providing login credentials, the drivers will recieve a one time password (OTP) to their mobile number. They will only be allowed to access the application once they have entered the OTP. 
+
+## Set up 
+
+1. [Download WSO2 Identity Server](https://wso2.com/identity-and-access-management/).
+
+2. Navigate to `<IS_HOME>/bin` directory via a command prompt and start the server by executing one of the following commands.
+
+    ``` java tab="Linux/MacOS"
+    sh wso2server.sh
+    ```
+
+    ``` java tab="Windows"
+    wso2server.bat run
+    ```
+
+3. Access the [management console]() and login using admin/admin credentials. 
```

Comment:
The link for management console is missing.

Reply:
Fixed 
485f165